sections/queueing_model/subsections/performance_measures/waiting_time/main.tex

```diff
@@ -0,0 +1,82 @@
+\subsubsection{Waiting time} \label{sec:waiting_time}
+
+Waiting time is the amount of time that individuals from either type wait in 
+waiting zone 1 so that they can receive their service. 
+For a given set of parameters there are three different performance measures 
+around the mean waiting time that can be calculated; the mean waiting time of
+type 1 individuals, the mean waiting time of type 2 individuals and the 
+overall mean waiting time. 
```

Comment:
Let us reorganise this section (and all the extra performance measure sections):

```suggestion
For a given set of parameters there are three different performance measures 
around the mean waiting time that can be calculated; the mean waiting time of
type 1 individuals:

\begin{equation}
% TODO Write expression here.
\end{equation}

The mean waiting time of type 2 individuals

\begin{equation}
% TODO Write expression here.
\end{equation}
 
The 
overall mean waiting time:

\begin{equation}
% TODO Write expression here.
\end{equation}
 
Where % TODO Write down what each expression/parameter is (if necessary).
```

Then move how you obtained this to an appendix.

Reply:
We then want to include here, some figures with a comparison with simulated and calculated values. 

This does require an introduction of the simulation model but that can be short again.

```
Appendix A gives more details on obtaining equations .... 
Figure ... shows a comparison of the calculated values (using equations ...) and simulated using a discrete event simulation (details of the discrete event simulation model are given in Appendix B).
```

That figure will be 3 violin plots I believe.

Reply:
The 3 violin plots will be the for: `type 1, type 2, overall` performance measures over multiple runs of the the DES model. On the same plot there will also be the value of the performance measure for the Markov model. Is that right?

Reply:
:+1: